um_canonicalize/tags.yaml

```diff
@@ -308,12 +351,21 @@ categories:
   Tense:
     - PRS
     - PST
+    - PST.FREQ
```

Comment:
Is this a past frequentative?

I don't see it mentioned in the Sylak-Glassman guidelines but frequentivity strikes me as modality, not tense.